### PR TITLE
APM > .NET > Add Npgsql integration

### PR DESCRIPTION
**Pull request by bobuva**
### What does this PR do?
Adds PostGreSQL integration to the lists of integrations for .NET Framework and .NET Core for the .NET Tracer.

### Motivation
Integration added in the .NET Tracer [1.14.0 release](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v1.14.0).

### Preview link

https://docs-staging.datadoghq.com/bob/npqsql-dotnet-tracer/tracing/setup/dotnet-framework#integrations

https://docs-staging.datadoghq.com/bob/npqsql-dotnet-tracer/tracing/setup/dotnet-core#integrations

(edit by @lucaspimentel: preview links)

### CODE REVIEW

File: content/en/tracing/setup/dotnet-core.md [hunk 1]
```diff
@@ -211,6 +211,7 @@ The .NET Tracer can instrument the following libraries automatically:
 | Redis (ServiceStack client)    | `ServiceStack.Redis` 4.0.48+                                            | `ServiceStackRedis`  |
 | Elasticsearch                  | `Elasticsearch.Net` 5.3.0+                                              | `ElasticsearchNet`   |
 | MongoDB                        | `MongoDB.Driver.Core` 2.1.0+                                            | `MongoDb`            |
+| PostgreSQL                     | `Npgsql` 4.0+                                                           | `Postgres`           |
```
Comment (lucaspimentel) [inline content/en/tracing/setup/dotnet-core.md]: The integration name (last column) is `AdoNet`. [See here.](https://github.com/DataDog/dd-trace-dotnet/blob/7fcc12ea86b59f5bc7027e060bd3d9ae37262ede/src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/NpgsqlCommandIntegration.cs#L16)

File: content/en/tracing/setup/dotnet-framework.md [hunk 2]
```diff
@@ -100,6 +100,7 @@ The .NET Tracer can instrument the following libraries automatically:
 | Redis (ServiceStack client)    | `ServiceStack.Redis` 4.0.48+   | `ServiceStackRedis`  |
 | Elasticsearch                  | `Elasticsearch.Net` 5.3.0+     | `ElasticsearchNet`   |
 | MongoDB                        | `MongoDB.Driver.Core` 2.1.0+   | `MongoDb`            |
+| PostgreSQL                     | `Npgsql` 4.0+                  | `Postgres`           |
```
Comment (lucaspimentel) [inline content/en/tracing/setup/dotnet-framework.md]: The integration name (last column) is `AdoNet`. [See here.](https://github.com/DataDog/dd-trace-dotnet/blob/7fcc12ea86b59f5bc7027e060bd3d9ae37262ede/src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/NpgsqlCommandIntegration.cs#L16)

File: content/en/tracing/setup/dotnet-core.md (hunk 1 above)
Comment (bobuva) [reply content/en/tracing/setup/dotnet-core.md]: Thanks -- didn't know that that was the intention of the column.

Comment (lucaspimentel) [reply content/en/tracing/setup/dotnet-core.md]: Users need to know the integration name to change settings like `DD_<INTEGRATION>_ENABLED` or `DD_<INTEGRATION>_ANALYTICS_ENABLED`. In code, they access integration settings through `TracerSettings.Integrations["<INTEGRATION>"]`.

**Comment by adamtobey** [comment]
@bobuva Hey, not sure if I'm the person you meant to request a review from. I'm not sure what this PR is 😄

**Pull request merged**